### ISSUE TITLE
provide ortholog links for priority fungal pathogens (and ideally make IDs searchable to reach pombe orthologs)

### ISSUE THREAD

**Issue by ValWood** [pombase/pombase-chado#1244]
WHO priority pathogens Oct 2022
source: https://iris.who.int/bitstream/handle/10665/363682/9789240060241-eng.pdf?sequence=1

| Priority Level      | Pathogen                                   | In PANTHER |
|---------------------|--------------------------------------------|-------|
| **Critical**        | *Cryptococcus neoformans*                 |    YES   |
| **Critical**        | *Candida auris*                           |       |
| **Critical**        | *Aspergillus fumigatus*                   |       |
| **Critical**        | *Candida albicans*                        |       |
| **High**            | *Nakaseomyces glabrata* (*Candida glabrata*) |       |
| **High**            | *Histoplasma spp.*                        |       |
| **High**            | Eumycetoma causative agents               |       |
| **High**            | *Mucorales* (e.g., agents causing mucormycosis) |       |
| **High**            | *Fusarium spp.*                           |       |
| **High**            | *Candida tropicalis*                      |       |
| **High**            | *Candida parapsilosis*                    |       |
| **Medium**          | *Scedosporium spp.*                       |       |
| **Medium**          | *Lomentospora prolificans*                |       |
| **Medium**          | *Coccidioides spp.*                       |       |
| **Medium**          | *Pichia kudriavzeveii* (*Candida krusei*) |       |
| **Medium**          | *Cryptococcus gattii*                     |       |
| **Medium**          | *Talaromyces marneffei*                   |       |
| **Medium**          | *Pneumocystis jirovecii*                  |       |
| **Medium**          | *Paracoccidioides spp.*                   |       |

**Comment by ValWood** [comment]
I put all priorities but we may restrict to Critical/high
1. Find out which are in PANTHER and request any exclusions
2. Use PANTHER to provide ortholog data store in Chado
3.  Link to UniPort entries in ortholog page section (open website ticket when ready)
4.  Make UniProt? IDs searchable in pombe simple search and gene mapper

**Comment by ValWood** [comment]
using PANTHER directly won't work here 
https://pantherdb.org/panther/speciesTree.jsp
because only includes species with a lot of data for transfer  in the core trees
We might be able to use PANTHER family IDs from InterPRO though?

**Comment by kimrutherford** [comment]
Is this related?:
 - pombase/website#2305

**Comment by ValWood** [comment]
yes, I'll close this. we won't be loading into chado (at least at this stage) but trying to link to a resource that covers the main pathogens.
